indico/htdocs/js/indico/modules/events/management.js

```diff
@@ -83,9 +83,12 @@
     };
 
     function refreshPersonFilters() {
+        $('.js-event-person-list #person-filters ul>li').removeClass('enabled');
```

Comment:
the first part of the selector isn't really needed since a `#id` selector is already unique